webapp/src/projects.tsx

```diff
@@ -430,6 +443,21 @@ export class ProjectsCarousel extends data.Component<ProjectsCarouselProps, Proj
                 this.hasFetchErrors = true;
             } else {
                 this.prevGalleries = pxt.Util.concat(res.map(g => g.cards));
+                const shuffle = this.props.shuffle
+                if (shuffle) {
+                    // keep last one
+                    const last = this.prevGalleries.pop();
```

Comment:
why?

Reply:
the last card a "add your game"
![image](https://user-images.githubusercontent.com/4175913/75823306-5b62e580-5d56-11ea-8ca9-a1d00101856d.png)